recipes/noc-reasoning-agent/scripts/utils/create_input_jsonl_from_incidents.py

```
@@ -0,0 +1,169 @@
+    "ran_vswr_alarm_resolution",
+    "ran_handover_failure_resolution",
+]
+
+
+def coerce_to_str(value) -> str:
+    if pd.isna(value):
+        return ""
+    # Ensure any non-string is converted to string safely
+    try:
+        return str(value)
+    except Exception:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)